spec/system/admins/company_values_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Kudos' do
+  let(:admin) { create(:admin) }
+
+  before do
+    sign_in admin
+  end
+
+  it 'All CRUD Company Values' do
+    visit('/admins')
+    click_link 'Admin Panel'
+    click_link 'Company Values'
+    click_link 'New Company Value'
+    fill_in 'Title', with: 'Company Value title'
+    click_button 'Create Company value'
+    expect(page).to have_content 'New Company Value was successfully created.'
```

Comment:
You are checking, that notification is displayed, but you are not checking, that Company Value has been created - I would check, if new Company Value is present on index page